tools/update_quest_data.py

```
@@ -197,7 +197,7 @@ def update_quest_enemies(region: str = "CN"):
                 free_quests.append(quest)
 
         if not free_quests:
-            print("无 Free 副本")
+            print(f"无 Free 副本")
```

Ruff (0.15.11)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)